README.md

```diff
@@ -1,12 +1,14 @@
 # mantis - Scala client for Ethereum Classic, The Daedalus Release
 
-In this release *Mantis* the new Ethereum Classic client produced by the [Grothendieck Team](https://iohk.io/projects/ethereum-classic/) 
-has been integrated with the [Daedalus](https://daedaluswallet.io/) wallet. This integration puts the Daedalus wallet management
-software into the hands of Ethereum Classic users, giving them a safe way to create, manage and backup their wallets.    
+The Daedalus release consists of two parts, the integration with the Daedalus Wallet and the release of the command line version
+ following on from the Beta release in August. This branch refers to release candidate 1 of the of the command line version.
  
-This version has been tested on Windows 10 and MacOS 
+The differences are very minor and relate only to how the RPC apis are enabled. In the command line version they are enabled over HTTP
+and in the Daedalus integration version they are forced to use HTTPS and client SSL certificate. The Daedalus version comes with a bundled JVM 
+but the command line version requires a JVM to be installed and available on the target machine.   
+    
+This version has been tested on Windows 10, MacOS and Ubuntu. 
```

Comment:
and Fedora :)

Reply:
(Changed it back to Linux, it's easier!)